### DISCUSSION TITLE
harbor作为helm包仓库，如何对接vela进行部署？

### DISCUSSION THREAD

**Discussion by SUMMERLm** [kubevela/kubevela#3321] [Q&A]
打通harbor下helm包基于vela自动化部署发布流程。
![image](https://user-images.githubusercontent.com/92348982/155290976-a87bdc14-407f-42b3-84e2-c5cea944bcdd.png)

![image](https://user-images.githubusercontent.com/92348982/155292377-b6a02b15-420d-4ec0-b7a8-bcb5d8eee537.png)
![image](https://user-images.githubusercontent.com/92348982/155292482-dc1b5dcf-3afa-4059-86fc-c11a6807be24.png)
![image](https://user-images.githubusercontent.com/92348982/155292529-b8d41a90-6508-4e2e-910f-cb559e51bc80.png)

**Comment by FogDong** [comment]
目前的 Trigger 支持的是 Harbor 中的镜像，所以会去更改 components 中的镜像。
如果是希望通过 Harbor 中的 Helm 包进行更新的话，需要去更改 helm component 中的 chart version。

目前社区暂时没有支持这个功能，如果您有需求的话，可以提个 issue。
有兴趣的话也可以来贡献一下，相关代码参考：https://github.com/oam-dev/kubevela/blob/master/pkg/apiserver/rest/usecase/webhook.go#L382

**Comment by FogDong** [comment]
目前的 Trigger 支持的是 Harbor 中的镜像，所以会去更改 components 中的镜像。
如果是希望通过 Harbor 中的 Helm 包进行更新的话，需要去更改 helm component 中的 chart version。

目前社区暂时没有支持这个功能，如果您有需求的话，可以提个 issue。
有兴趣的话也可以来贡献一下，相关代码参考：https://github.com/oam-dev/kubevela/blob/master/pkg/apiserver/rest/usecase/webhook.go#L382
